src/app/widgets/cat-source/cat-source-3fgl/cat-source-3fgl.component.html

```diff
@@ -1,3 +1,14 @@
 <p>
   This is CatSource3FGLComponent
 </p>
+
+<select *ngIf="catalog" (change)="onSelect($event.target.value)">
+  <option value="">Choose a source</option>
+  <option *ngFor="let source of catalog.data" value="{{source.data.id}}">{{source.data.Source_Name}}</option>
+</select>
+
+<div *ngIf="selection">
+  <h3>{{catalog.data[selection].data.Source_Name}}</h3>
```

Comment:
Is there a way to introduce variables in templates.
I.e. instead of repeating `catalog.data[selection].data` 50 times, put `var d =catalog.data[selection].data`